tests/org.jboss.reddeer.ui.test/src/org/jboss/reddeer/ui/test/wizard/impl/RedDeerTestPluginWizard.java

```diff
@@ -27,6 +27,6 @@
 	 * Default constructor
 	 */
 	public RedDeerTestPluginWizard() {
-		super(CATEGORY, NAME);
+		super("New RedDeer Test Plugin",CATEGORY, NAME);
```

Comment:
also just a though. Now when all Wizard/Wizard dialogs have shell title, maybe we could have a unified approach for this. Some dialogs uses constant of a class, others place a String directly.